core/server/data/schema/default-settings.json

```diff
@@ -590,6 +590,13 @@
         "editor_default_email_recipients_filter": {
             "defaultValue": "all",
             "type": "string"
+        },
+        "editor_is_launch_complete": {
+            "defaultValue": "false",
+            "validations": {
```

Comment:
Other boolean settings we have also have the `"isEmpty": false` validation - is it worth adding here?

Reply:
Yea good point, no reason to not add it here, updating